bin/generator.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Used to start the generator 
+
+# error prints an error to STDERR and exits.
+
+if [ -z "$1" ] ; then
+    echo "usage: ./bin/template.sh exercise-slug "
+    exit 1
+fi
+
+slug="$1"
+
+cat "./exercises/practice/${slug}/.meta/test_template.ecr" > "./bin/templates/template.ecr"
```

Comment:
Shouldn't this be the other way around?

```suggestion
cat "./bin/templates/template.ecr" > "./exercises/practice/${slug}/.meta/test_template.ecr"
```

Reply:
No it should be that way, the script takes the template for each exercise and move it to a set folder.